core/server/controllers/frontend/channel-config.js

```diff
@@ -14,13 +13,13 @@ channelConfig = function channelConfig() {
             name: 'tag',
             route: '/' + config.routeKeywords.tag + '/:slug/',
             postOptions: {
-                filter: labs.isSet('hiddenTags') ? 'tags:\'%s\'+tags.hidden:false' : 'tags:\'%s\''
+                filter: 'tags:\'%s\''
```

Comment:
Hmm... I just tested and your comment on the code below is correct - it doesn't get updated properly - that's weird I'm sure I tested that!